ape_safe/accounts.py

```diff
@@ -372,6 +372,48 @@ def create_safe_tx(self, txn: Optional[TransactionAPI] = None, **safe_tx_kwargs)
         }
         return self.safe_tx_def(**safe_tx)
 
+    def all_delegates(self) -> Iterator[AddressType]:
+        for delegates in self.client.get_delegates().values():
+            yield from delegates
+
+    def propose_safe_tx(
```

Comment:
the `propose` cli method could probably be refactored to utilize this now

Reply:
Looked into this, but seems more complicated than I'm willing to deal with now

Think I will try to refactor that CLI method later